azure-keyvault/azure/keyvault/secrets/_client.py

```diff
@@ -149,7 +179,8 @@ def set_secret(
         }
 
         attributes = _SecretAttributes(enabled=enabled, not_before=not_before, expires=expires)
-        secret = SecretSetParameters(secret_attributes=attributes, value=value, tags=tags, content_type=content_type)
+        secret = SecretSetParameters(
+            secret_attributes=attributes, value=value, tags=tags, content_type=content_type)
```

Comment:
Was this line split by a formatter? Our `pylintrc` specifies 120 for max line length.